src/ObjCommon/Game/IW4/Leaderboard/JsonLeaderboardDef.h

```diff
@@ -12,6 +12,8 @@
 
 namespace IW4
 {
+    static constexpr auto LEADERBOARD_DEF_VERSION = 1u;
```

Comment:
i dont think its necessary to introduce a constant in ObjCommon for this. The reason is that the loader will probably have to support multiple versions for backwards compatibility while the dumper will only dump the latest version.
So these two parts are kind of not necessarly connected in that regard.